packages/main/src/TabContainer.js

```diff
@@ -255,6 +255,15 @@ const metadata = {
 			type: Object,
 			multiple: true,
 		},
+
+		_tabItems: {
+			type: Object,
+			multiple: true,
+		},
+
+		selectedTab: {
```

Comment:
Is it necessary to add a `selectedTab` property, when there already exists `_selectedTab`?